definitions/region/model_native_regions/image_v3.2.yaml

```diff
@@ -2,4 +2,28 @@
 - IMAGE 3.2:
   - IMAGE 3.2|WEU
   - IMAGE 3.2|CEU
-  
+  - IMAGE 3.2|BRA
+  - IMAGE 3.2|CAN 
+  - IMAGE 3.2|CHN 
+  - IMAGE 3.2|EAF 
+  - IMAGE 3.2|INDIA 
+  - IMAGE 3.2|INDO
+  - IMAGE 3.2|JAP
+  - IMAGE 3.2|KOR 
+  - IMAGE 3.2|ME 
+  - IMAGE 3.2|MEX
+  - IMAGE 3.2|NAF 
+  - IMAGE 3.2|OCE
+  - IMAGE 3.2|RCAM
+  - IMAGE 3.2|RSAF
+  - IMAGE 3.2|RSAM
+  - IMAGE 3.2|RSAS
+  - IMAGE 3.2|RUS
+  - IMAGE 3.2|SAF
+  - IMAGE 3.2|SEAS
+  - IMAGE 3.2|STAN
+  - IMAGE 3.2|TUR
+  - IMAGE 3.2|UKR
+  - IMAGE 3.2|USA
+  - IMAGE 3.2|WAF
+  - IMAGE 3.2|World

```

Comment:
There is no need a model-specific "World" region, you can just use the common "World" region for global data.
```suggestion

```